### DISCUSSION TITLE
关联项目aliyundrive-fuse无反馈渠道，请求更新官方接口

### DISCUSSION THREAD

**Discussion by juneix** [messense/aliyundrive-webdav#758] [Q&A] [answered]
### 问题描述

之前因为阿里云盘关闭非官方接口的原因，导致aliyundrive-webav和关联项目aliyundrive-fuse无法使用。目前来看，aliyundrive-webdav已经更换为官方接口了，请问aliyundrive-fuse能同步修复一下吗？这个工具用来挂载plex影音库非常好用。

### 重现步骤

1. aliyundrive-webdav 2.0.4 已更换为官方接口，可正常使用
2. aliyundrive-fuse 0.1.14 尚未同步修复接口问题

### 版本

2.0.4

### 运行平台

OpenWrt

### 日志

无

**Comment by messense** [comment, marked as answer]
暂无修复计划。

**Comment by 123385918** [comment]
@juneix 请问现在解决了吗?还是用aliyundrive-webav+rclone挂载的plex影视库?

**Reply by juneix** [reply]
我已经换成clouddrive2了，基于阿里云盘官方开放平台，配合plex使用一切正常，但它是闭源的。供参考

**Reply by 303922010** [reply]
不太会用，我也换成alist了。。。。

**Reply by OpenSourcerX** [reply]
clouddrive2在Openwrt中自动挂载有问题，还是aliyundrive-fuse好用，期待作者更新。
